tnt/src/main/java/org/dcsa/conformance/standards/tnt/party/AttributeMapping.java

```diff
@@ -0,0 +1,160 @@
+package org.dcsa.conformance.standards.tnt.party;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.Getter;
+
+import java.time.OffsetDateTime;
+import java.util.*;
+import java.util.function.BiPredicate;
+
+@Getter
+public class AttributeMapping {
+  private final String jsonPath;
+  private final BiPredicate<JsonNode, String> condition;
+  private final Set<String> values;
+  private static final String DOCUMENT_TYPE_CODE = "documentTypeCode";
+  private static final String EVENT_CREATED_DATE_TIME = "eventCreatedDateTime";
+
+  public AttributeMapping(String jsonPath, BiPredicate<JsonNode, String> condition, Set<String> values) {
+    this.jsonPath = jsonPath;
+    this.condition = condition;
+    this.values = values;
+  }
+
+  public static Map<String, List<AttributeMapping>> initializeAttributeMappings() {
```

Comment:
This method actually generates the attribute mappings, not only the init, but it actually returns it. That should be reflected in the name, such as getAttributeMappings?